src/headers.js

```diff
@@ -12,10 +12,9 @@ function normalizeAndValidateHeaderName (name) {
 }
 
 function normalizeAndValidateHeaderValue (name, value) {
-  const normalizedHeaderName = normalizeAndValidateHeaderName(name)
   const normalizedHeaderValue = value.replace(/^[\n\t\r\x20]+|[\n\t\r\x20]+$/g, '')
-  validateHeaderValue(normalizedHeaderName, normalizedHeaderValue)
-  return [normalizedHeaderName, normalizedHeaderValue]
+  validateHeaderValue(name, normalizedHeaderValue)
```

Comment:
Wasn't sure if `validateHeaderValue` required a normalized/validated headerName or not. I'll see what its doing under-the-hood

Reply:
It doesn't use it. It's just to give a name in the thrown exception.